examples/tree_2d_dgsem/elixir_euler_warm_bubble.jl

```diff
@@ -0,0 +1,126 @@
+using OrdinaryDiffEq
+using Trixi
+
```

Comment:
Similar comments for this elixir. The reference values and domain size do not match that of docstring reference or the original setup from Oswald's branch. Some clarification is needed here.

Reply:
For TreeMesh the domain size indeed differs from the reference as we are only able to create square domains.

Reply:
Maybe add this as a comment to the elixir?

Reply:
Done!